modules/orbit/pom.xml

```diff
@@ -66,6 +66,29 @@
             <artifactId>bcprov-jdk15on</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.opensaml</groupId>
+            <artifactId>opensaml1</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.opensaml</groupId>
+            <artifactId>opensaml</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.opensaml</groupId>
+            <artifactId>xmltooling</artifactId>
+            <version>1.4.6</version>
```

Comment:
shall we  put properties or either refer by parent pom

Reply:
Moved into properties.